Command/InstallFontCommand.php

```diff
@@ -19,10 +19,12 @@
 use Symfony\Component\Filesystem\Filesystem;
 use Symfony\Component\Finder\Finder;
 use Symfony\Component\HttpKernel\KernelInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
```

Comment:
Should be alpha sorted, that's why CS checks are failing

Reply:
Changed the sort order, should work now